packages/studio-ui/src/web/components/Layout/BottomPanel/Logs/index.tsx

```diff
@@ -107,14 +115,27 @@ class BottomPanel extends React.Component<Props, State> {
     })
   }
 
+  renderLogContent(content: string) {
+    const isHTML = (str: string) => {
+      const doc = new DOMParser().parseFromString(str, 'text/html')
+      return Array.from(doc.body.childNodes).some((node) => node.nodeType === 1)
+    }
+
+    if (isHTML(content)) {
+      return <span className={logStyle.message} dangerouslySetInnerHTML={{ __html: content }} />
+    } else {
```

Comment:
Why do we want do dangerously do anything with user provided input? Doesn't that make use vulnerable to XSS

Reply:
I personally have no clue why but since the code was there before, we can consider that this PR does not bring any vulnerabilities.

Reply:
I think this is a major vulnerability. Someone in the webchat could send a script tag as a message, then some server admin could check the logs and have a random script executing in their browser. This script could steal their login token and give root access to the hacker. Who's asking for this feature?